tests/e2e.bats

```diff
@@ -175,3 +194,24 @@
   [ "$status" -eq 0 ]
   [[ "$output" == "dummy_bpf.o:[classifier]" ]]
 }
+
+
+@test "verify resourceslices are cleaned up after driver removed" {
+  # Patch the DaemonSet with a dummy nodeSelector to disable it
+  run kubectl patch ds "$ds_name" -n "$namespace" \
+    --type='merge' \
+    -p="{\"spec\": {\"template\": {\"spec\": {\"nodeSelector\": {\"$node_selector_key\": \"$node_selector_value\"}}}}}"
+  [ "$status" -eq 0 ]
+
+  # Wait for pods to be deleted
+  run kubectl wait --for=delete pod -l app=dranet -n "$namespace" --timeout=60s
+  [ "$status" -eq 0 ]
+
+  # Confirm no DraNet pods left
+  run kubectl get pods -n "$namespace" -l app=dranet --no-headers
+  [ "$status" -ne 0 ] || [ -z "$output" ]
+
+  # Confirm ResourceSlices are gone
+  run kubectl get resourceslices -A --no-headers
```

Comment:
ResourceSlices are cluster scoped resources. We do not need `-A` which lists across all namespaces.